test/dynamo/test_repros.py

```diff
@@ -1015,6 +1015,8 @@ def test_create_rand_mask_from_inputs(self):
         self.assertEqual(cnt.frame_count, 1)
         self.assertEqual(cnt.op_count, 8)
 
+    # TODO: make set_rng_state work with fake tensor
+    @patch.object(torch._dynamo.config, "fake_tensor_propagation", False)
```

Comment:
Xfail it, rather than skip please, so we know when its fixed (unexpected success)

Reply:
I tried to add `@unittest.expectedFailure`, but what makes it complicated is we have `test_no_fake_tensors.py` which patches `torch._dynamo.config. fake_tensor_propagation` to `False` and makes this test unexpectedly succeed. I tried to rewrite the decorator with a conditional checking but that doesn't work well with `object.patch`. Given the unlikelihood this test will unexpectedly get fixed, I will skip adding xfail here.